tests/toc.spec.ts

```diff
@@ -4,7 +4,7 @@ test('table of contents redirects to proper item', async ({ page }) => {
   await page.goto('/basics/getting-started/record-your-app')
 
   const toc = page.locator('[data-testid="table-of-contents"]')
-  await toc.getByText('Inspect your replay').click()
+  await toc.getByText('Inspect your recording').click()
```

Comment:
This is a bit odd because the page title this `click` event will navigate to and the link text on this element don't match anymore, but I _still_ think it is clearer to the user now than it was before.

Reply:
Lets use replay

Reply:
Got it